<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetLicense</class>
 <widget class="QWidget" name="GetLicense">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Get License</string>
  </property>
  <widget class="QLineEdit" name="enterName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterLName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Last Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CheckBttn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="ExpirationDate">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PoliceCheckPIC">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/PoliceCheck/immigration_check_officer_security_passport-512.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Exiration Date is:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="PoliceCheck.qrc"/>
 </resources>
 <connections/>
</ui>
